content/ಅಂಕಿಗಳ ವಿಭಕ್ತಿ.md

```diff
@@ -0,0 +1,97 @@
+ಅದರಲ್ಲೂ ಹನ್ನೊಂದರಿಂದ ಹದಿನೆಂಟರವರೆಗಿನವು ಬೇರೆಯವುಗಳಂತಲ್ಲ.
+
+ ಹಿಂದಿ ಸಂಸ್ಕೃತ ಭಾಷೆಗಳಲ್ಲಿ 
+11 = ಗ್ಯಾರಹ್ = ಏಕಾದಶ (1 +10)
+12 = ಬಾರಹ್ = ದ್ವಾದಶ (2 +10)
+13 = ತೇರಹ್ = ತ್ರಯೋದಶ (3+10)
+14 = ಚೌದಹ್ = ಚತುರ್ದಶ (4+10)
+ಇತ್ಯಾದಿ ಏಕರೂಪದಲ್ಲಿ ಇವೆ, ಆದರೆ ಕನ್ನಡದ ಹನ್ನೊಂದು ಹನ್ನೆರಡು ಹದಿಮೂರು ಇತ್ಯಾದಿಗಳಲ್ಲಿ `10+1,2,3` ರೀತಿಯಲ್ಲಿ ಇರುವುದು ಮಾತ್ರವಲ್ಲದೇ ಹತ್ತರಭಾಗವೂ ಏಕರೂಪವಾಗಿಲ್ಲ, `ಹನ್ನ` ಮತ್ತು `ಹದಿ` ಎಂಬ ಎರಡು ರೂಪಗಳು ಕಾಣುತ್ತವೆ. ಹಾಗಾದರೆ ಕನ್ನಡ ಭಾಷೆಯ ಸಂಖ್ಯೆಗಳಿಗೊಂದು ತಾರ್ಕಿಕ ವ್ಯವಸ್ಥೆಯ ಮೂಲವಿಲ್ಲವೇ? ಆಡುಮಾತಿನಿಂದ ಬಂದುದೇ? ಹಾಗಲ್ಲ ಎಂದು ತೋರಿಸುವುದು ಈ ಲೆಖನದ ಉದ್ದೇಶ.
+
+ಈ ಸಂಖ್ಯೆಗಳ ಹುಟ್ಟನ್ನು ಹಳಗನ್ನಡದ ಹಿನ್ನೆಲೆಯಿಟ್ಟುಕೊಂಡು ನೋಡಿದಾಗ 11-18 ರಲ್ಲಿ ಹುದುಗಿರುವ **ಪಂಚಮೀ ವಿಭಕ್ತಿ* ಯು ನಮ್ಮಲ್ಲಿ ಮಾತ್ರ ಇರುವ ಸ್ವಾರಸ್ಯ ಕಾಣುತ್ತದೆ. ಉಳಿದ 19ರ ಮೇಲಿನ ಸಂಖ್ಯೆಗಳ ಹಿಂದಿನ ತಾರ್ಕಿಕ ವ್ವವಸ್ಥೆಯನ್ನೂ ನೊಡೋಣ.
+
+-ಗಣೇಶಕೃಷ್ಣ ಶಂಕರತೋಟ
+## ಪರವಾನಗಿ / License
+
+[![GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007](https://www.gnu.org/graphics/gplv3-127x51.png) ![Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License](https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png)](../README.md#ಪರವಾನಗಿ--license)
+
+## ಹದಿಮೂರು
+ಈ ಅಧ್ಯಯನವನ್ನು ಹದಿಮೂರರಿಂದ ಆರಂಭಿಸೋಣ. ಹನ್ನೊಂದು ಹನ್ನೆರಡುಗಳು ಯಾಕಲ್ಲವೆಂದು ಮುಂದೆ ವಿಷದವಾಗುತ್ತದೆ.
+ಹದಿಮೂರರಲ್ಲಿ ಇರುವ `ಹತ್ತು` ಮತ್ತು `ಮೂರು` ಎಂಬ ಎರಡು ಭಾಗಗಳನ್ನು ನೋಡಬೇಕು. ಅವುಗಳ ಮಧ್ಯೆ ಇರುವ ಸಂಬಂಧ ನೋಡಬೇಕು.
+ಹತ್ತರ ಮೂರೇ, ಹತ್ತು ಮತ್ತು ಮೂರೇ, ಹತ್ತರ ನಂತರ ಮೂರೇ ?
+
+### ಹತ್ತು
+ನಾಮಪದಗಳ ನಡುವಿನ ಸಂಬಂಧವನ್ನು ನಾವು ವಿಭಕ್ತಿಯಿಂದ ತೋರಿಸುತ್ತೇವೆ. ಪ್ರತ್ಯಯವನ್ನು ನೋಡಿ ವಿಭಕ್ತಿಯನ್ನು ತಿಳಿದು ಒಂದು ನಾಮ ಇನ್ನೊಂದರ ಮೇಲೆ ಹೇಗೆ ಅವಲಂಬಿಸಿದೆ ಎಂದು ಅರಿಯಬೇಕು. ಹದಿಮೂರರ ಹುಟ್ಟು ಹಳಗನ್ನಡದಲ್ಲಿ ಆದದ್ದು. 10 = ಪತ್ತು. ಪತ್ತು ಶಬ್ದಕ್ಕೆ ವಿಭಕ್ತಿ ಪ್ರತ್ಯಯಗಳನ್ನು ಜೋಡಿಸಿದಾಗ ಕೆಳಗಿನ ಪದಗಳು ಉಂಟಾಗುತ್ತವೆ
+1. ಪತ್ತು
+2. ಪತ್ತಂ (ಹತ್ತನ್ನು)
+3. ಪತ್ತಿಂ (ಹತ್ತನ್ನು ಬಳಸಿ)
+4. ಪತ್ತಕ್ಕೆ (ಹತ್ತಕ್ಕೆ)
+5. ಪತ್ತಿನತ್ತಣಿಂ,ಪತ್ತರದೆಸೆಯಿಂದ, ಪತ್ತಿಂ (ಹತ್ತಿಂದ, ಹತ್ತರಿಂದ)
+6. ಪತ್ತರ (ಹತ್ತರ)
+7. ಪತ್ತೊಳ್ (ಹತ್ತಲ್ಲಿ)
+
+ಇವುಗಳಲ್ಲಿ ಕೆಲವು ಪದಗಳು ಉಪಯೋಗಕ್ಕಿಲ್ಲ. ನಮಗೆ ಅವಷ್ಯವಾಗಿರುವವು 3,5, 6 ನೇ ವಿಭಕ್ತಿಗಳು.
+### ತೃತಿಯೆಯೂ ಪಂಚಮಿಯೂ
+`ಕತ್ತಿಯಿಂ ಕಡಿದಂ`, `ಕತ್ತಿಯಿಂದ ಕಡಿದನು` ಎಂಬಲ್ಲಿ ಕತ್ತಿ ಉಪಕರಣ
+`ಕತ್ತಿಯೊಳ್ ಕಡಿದಂ`, `ಕತ್ತಿಯಲ್ಲಿ ಕಡಿದನು` ಎಂದಾಗಲೂ ಅಷ್ಟೆ, ಕತ್ತಿ ಉಪಕರಣ. ಪ್ರತ್ಯಯ ಏನೆ ಇದ್ದರೂ ವಿಭಕ್ತಿ ಕರಣಕಾರಕದ ತೃತಿಯಾ ವಿಭಕ್ತಿ.
+`ಮರದಿಂ ಫಲ ಬೀೞ್ದುದು`, `ಮರದಿಂದ ಹಣ್ಣು ಬಿದ್ದುದು`
+`ಗುಡ್ಡದಿಂದ ಆಚೆ`, `ಈ ಕೋಲಿಂದ ಉದ್ದ` ಎಂಬಲ್ಲಿ **ಇಂ, ಇಂದ** ಪ್ರತ್ಯಯಗಳಿದ್ದರೂ `ಮರ`, `ಗುಡ್ಡ`, `ಕೋಲು`ಗಳು ಉಪಕರಣಗಳಲ್ಲ ಎಂಬುದು ಸ್ವಯಂ ವೇದ್ಯ. ಮರವಿದ್ದಲ್ಲಿ ಇದ್ದ ಹಣ್ಣು ಈಗ ಇಲ್ಲ. ಇದನ್ನು ಅಪಾದಾನ (ಒಂದು ವಸ್ತುವಿನಿಂದ ದೂರ ಸರಿಯುವುದು) ಕಾರಕ ಎನ್ನುತ್ತಾರೆ, ಇದುವೇ ಪಂಚಮೀ ವಿಭಕ್ತಿಯ . ಸೇಡಿಯಾಪು ಕೃಷ್ಣ ಭಟ್ಟರು ಪಂಚಮೀ ವಿಭಕ್ತಿ ಲೇಖನದಲ್ಲಿ ಈ ವಿಚಾರವನ್ನು ಪ್ರತಿಪಾದಿಸಿದ್ದಾರೆ[1]. `ಇಂ` ಮತ್ತು `ಇಂದ` ಪ್ರತ್ಯಯಗಳು ಮೂಲತಹ ಪಂಚಮೀ ವಿಭಕ್ತಿಗಳು ಎಂಬುದನ್ನು ಸಾಧಿಸಿ ತೋರಿಸಿದ್ದಾರೆ.`ಅಲ್ಲಿಂದ` ಎಂಬ ಶಬ್ದಕ್ಕೆ `ಅಲ್ಲಿ-ಗೆ` ಸಾಪೇಕ್ಷವಾಗಿ ಎಂಬ ಪಂಚಮೀ ಅರ್ಥವೇ ಹೊರತು `ಅಲ್ಲಿ-ಯನ್ನುಪಯೋಗಿಸಿ` ಎಂಬ ತೃತಿಯಾರ್ಥ ಬರುವುದೇ ಇಲ್ಲ.
+
+**ಹತ್ತು ಇದ್ದಲ್ಲಿಂದ**, **ಹತ್ತಕ್ಕೆ ಸಾಪೇಕ್ಷವಾಗಿ** ಎಂಬ ಅರ್ಥಬರಲು ಪಂಚಮೀ ವಿಭಕ್ತಿ ಮತ್ತು ಅದರ ಪ್ರತ್ಯಯವನ್ನು ಪ್ರಯೋಗಿಸಬೇಕೆಂಬುದನ್ನು ನೋಡಿದೆವು. ಪಂಚಮೀ ಪ್ರತ್ಯಯವಾಗಿ `-ದೆಸೆಯಿಂದ`, `-ಅತ್ತಣಿಂ` ಗಳನ್ನು ನಾವು ಎಲ್ಲೂ ಪ್ರಯೋಗಿಸುವುದಿಲ್ಲ, **ಇಂ, ಇಂದ** ಗಳನ್ನೇ ಉಪಯೊಗಿಸುವುದು ಎಂದೂ ನೋಡಿದೆವು.ಆದ್ದರಿಂದ **ಪತ್ತಿಂ** ಶಬ್ದವು **ಪತ್ತರತ್ತಣಿಂ** **ಹತ್ತರದೆಸೆಯಿಂದ** ಕ್ಕೆ ಸಮಾನವಾದ ಪಂಚಮೀ ಪ್ರಯೋಗವೆಂಬುದು ನಿಸ್ಸಂಶಯ.
```

Comment:
Mention `ಹತ್ತರಿಂದ` as another translation.

Also, hyperlink https://github.com/amshuman-kr/kannadada-bhakti/blob/main/content/ಸರ್ವನಾಮಗಳ%20ಸರ್ಗ/ಸರ್ವನಾಮಗಳ%20ಸರ್ಗ.md#ಕನ್ನಡದ-ಪ್ರಥಮಪುರುಷ-ಸರ್ವನಾಮಗಳ-ವಿಭಕ್ತಿಪ್ರತ್ಯಯಗಳ-ನಂಟು somewhere in this line? Shameless Plug™️

Reply:
There are multiple places I need to Link to your pages. for now I have copied some references, but really they should be links. will do.